### ISSUE TITLE
Actions not showing label (cut off)

### ISSUE THREAD

**Issue by BigRoy** [pyblish/pyblish-qml#175]
# Issue

The labels of Actions that are a bit longer get cut off in the context menu.

See:

![action_not_showing_full_label](https://cloud.githubusercontent.com/assets/2439881/14528650/e171f4d2-0250-11e6-87b0-c2694c7c7f21.jpg)

# Goal

I think the context menu should resize to a wider area as the label is longer.

**Comment by tokejepsen** [comment]
I've noticed the context menu also doesn't go beyond the borders of the window. Don't know whether that is even possible.

**Comment by BigRoy** [comment]
> I've noticed the context menu also doesn't go beyond the borders of the window. Don't know whether that is even possible.

Had a quick look here and it seems you're right, same here. That also feels a bit odd. ;)

**Comment by mottosso** [comment]
> I've noticed the context menu also doesn't go beyond the borders of the window.

Yeah, it's a limitation of how I developed the menu. It's possible to make it its own window, like normal tooltips and context menus are. Pull requests are welcome. :)
